### ISSUE TITLE
Install fails on OSX 10.10

### ISSUE THREAD

**Issue by volopasse** [astrada/google-drive-ocamlfuse#129]
Similar to [#40](https://github.com/astrada/google-drive-ocamlfuse/issues/40), I'm trying to install on OSX 10.10. Install fails with a very similar pattern - looks like OSXFUSE is not being linked up correctly. Both FUSE and opam were installed via Homebrew if that is relevant

```
=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] The compilation of ocamlfuse failed at "make -C lib
        INCDIRS=/Users/Denis/.opam/system/lib/camlidl".
Processing  1/2: [ocamlfuse: ocamlfind remove]
#=== ERROR while installing ocamlfuse.2.7.1-cvs2 ==============================#
# opam-version 1.2.1
# os           darwin
# command      make -C lib INCDIRS=/Users/Denis/.opam/system/lib/camlidl
# path         /Users/Denis/.opam/system/build/ocamlfuse.2.7.1-cvs2
# compiler     system (4.02.1)
# exit-code    2
# env-file     /Users/Denis/.opam/system/build/ocamlfuse.2.7.1-cvs2/ocamlfuse-38445-e7966f.env
# stdout-file  /Users/Denis/.opam/system/build/ocamlfuse.2.7.1-cvs2/ocamlfuse-38445-e7966f.out
# stderr-file  /Users/Denis/.opam/system/build/ocamlfuse.2.7.1-cvs2/ocamlfuse-38445-e7966f.err
### stdout ###
# camlidl -header  \
# [...]
# making ._ncdi/Fuse_bindings.di from Fuse_bindings.mli
# making ._d/Fuse_bindings.d from Fuse_bindings.ml
# making ._d/Fuse.d from Fuse.ml
# making ._d/Fuse_lib.d from Fuse_lib.ml
# making ._d/Unix_util.d from Unix_util.ml
# making ._d/Result.d from Result.ml
# ocamlc -c -cc "cc" -ccopt "-fPIC -D_FILE_OFFSET_BITS=64 -I. -pthread \
#               -DPIC -DNATIVE_CODE   \
#                 -I/Users/Denis/.opam/system/lib/camlidl  -o Fuse_bindings_stubs.o " Fuse_bindings_stubs.c 
### stderr ###
# In file included from Fuse_bindings_stubs.c:17:
# ./Fuse_bindings.h:16:10: fatal error: 'fuse.h' file not found
# #include <fuse.h>
#          ^
#1 error generated.
# make[1]: *** [Fuse_bindings_stubs.o] Error 2
# make: *** [native-code-library] Error 2



=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
The following actions were aborted
  ∗  install google-drive-ocamlfuse 0.5.17
The following actions failed
  ∗  install ocamlfuse 2.7.1-cvs2
No changes have been performed
```

**Comment by astrada** [comment]
Where does homebrew install libfuse headers (fuse.h)?

You should put that dir in CPPFLAGS (overriding the default location) to correctly install ocamlfuse like this:

    $ CPPFLAGS='-DFUSE_USE_VERSION=26 -I/usr/local/include/osxfuse' opam install ocamlfuse

**Comment by AndreTheHunter** [comment]
I still get an error even when I set the CPPFLAGS (notice the `-DFUSE_USE_VERSION=26 -I/usr/local/include/osxfuse`).
```
The following actions will be performed:
  ∗  install ocamlfuse 2.7.1-cvs2

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ocamlfuse] Archive in cache

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] The compilation of ocamlfuse failed at "make -C lib INCDIRS=/Users/andre.dejager/.opam/system/lib/camlidl".
Processing  1/1: [ocamlfuse: ocamlfind remove]
#=== ERROR while installing ocamlfuse.2.7.1-cvs2 ==============================#
# opam-version 1.2.2
# os           darwin
# command      make -C lib INCDIRS=/Users/andre.dejager/.opam/system/lib/camlidl
# path         /Users/andre.dejager/.opam/system/build/ocamlfuse.2.7.1-cvs2
# compiler     system (4.02.3)
# exit-code    2
# env-file     /Users/andre.dejager/.opam/system/build/ocamlfuse.2.7.1-cvs2/ocamlfuse-24062-6d07ae.env
# stdout-file  /Users/andre.dejager/.opam/system/build/ocamlfuse.2.7.1-cvs2/ocamlfuse-24062-6d07ae.out
# stderr-file  /Users/andre.dejager/.opam/system/build/ocamlfuse.2.7.1-cvs2/ocamlfuse-24062-6d07ae.err
### stdout ###
# [...]
# making ._d/Fuse.d from Fuse.ml
# making ._d/Fuse_lib.d from Fuse_lib.ml
# making ._d/Unix_util.d from Unix_util.ml
# making ._d/Result.d from Result.ml
# ocamlc -c -cc "cc" -ccopt "-fPIC -D_FILE_OFFSET_BITS=64 -I. -pthread \
# 				-DPIC -DNATIVE_CODE -DFUSE_USE_VERSION=26 -I/usr/local/include/osxfuse  \
# 				  -I/Users/andre.dejager/.opam/system/lib/camlidl  -o Fuse_bindings_stubs.o " Fuse_bindings_stubs.c 
# ocamlc -c -cc "cc" -ccopt "-fPIC -D_FILE_OFFSET_BITS=64 -I. -pthread \
# 				-DPIC -DNATIVE_CODE -DFUSE_USE_VERSION=26 -I/usr/local/include/osxfuse  \
# 				  -I/Users/andre.dejager/.opam/system/lib/camlidl  -o Unix_util_stubs.o " Unix_util_stubs.c 
### stderr ###
# Unix_util_stubs.c:30:10: fatal error: 'sys/vfs.h' file not found
# #include <sys/vfs.h>
#          ^
# 1 error generated.
# make[1]: *** [Unix_util_stubs.o] Error 2
# make: *** [native-code-library] Error 2

=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
The following actions failed
  ∗  install ocamlfuse 2.7.1-cvs2
No changes have been performed

=-=- ocamlfuse.2.7.1-cvs2 troobleshooting -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
=> This package relies on external (system) dependencies that may be missing. `opam depext ocamlfuse.2.7.1-cvs2' may help you find the correct installation for your system.
```

**Comment by astrada** [comment]
It looks like OSX does not have the header file `sys/vfs.h` (see https://github.com/kartverket/fyba/issues/12). Maybe it can be replaced by `sys/mount.h`. Unfortunately I don't have an OSX box, so I don't have a way to test possible workarounds.

**Comment by astrada** [comment]
Should be fixed by astrada/ocamlfuse#3.
